src/frontend/src/lib/components/buy/BuyModal.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+	import { Modal } from '@dfinity/gix-components';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+</script>
+
+<Modal on:nnsClose={modalStore.close}>
+	<svelte:fragment slot="title"
+		><p class="text-xl">
```

Comment:
Shouldn't `<p class="text-xl">` be removed here?

Reply:
Good catch! Forgot to remove it!